scripts/dats_validator/validator.py

```diff
@@ -105,11 +106,54 @@ def validate_extra_properties(dataset):
                         f"{[k for k in REQUIRED_EXTRA_PROPERTIES.keys()]}")
 
 
+def validate_formats(dataset):
+    """ Checks if the values in the formats field of the JSON object follows the upper case convention without dots. """
+
+    errors_list = []
+    format_exceptions = ['BigWig', 'NIfTI', 'RNA-Seq']
+
+    # check that distributions have a formats property as this is required in the schema
+    for distribution_dict in dataset['distributions']:
+        if 'formats' not in distribution_dict.keys():
+            error_message = f"Validation error in {dataset['title']}: distributions." \
+                            f"formats - 'formats' property is missing under distributions. " \
+                            f"Please add a the 'formats' property to 'distributions'."
+            errors_list.append(error_message)
+
+    if errors_list:
+        return False, errors_list
```

Comment:
@cmadjar The function returns `False` and `errors_list` in two places, this is redundant. You can rearrange the if condition. First, you check if 'formats' is not there, then you can add the rest in else. 

for example:

```
format_exceptions = ['BigWig', 'NIfTI', 'RNA-Seq']
for distribution_dict in dataset['distributions']:
    if 'formats' not in distribution_dict:
        error_message = f"Validation error in {dataset['title']}: distributions." \
                                    f"formats - 'formats' property is missing under distributions. " \
                                    f"Please add a the 'formats' property to 'distributions'."
        errors_list.append(error_message)
    else:
        for file_format in distribution_dict['formats']:
            if file_format != file_format.upper() and file_format not in format_exceptions:

            ...... etc. ........

if errors_list:
    return False, errors_list
else:
    return True, errors_list
```